lib/scrabble.rb

```diff
@@ -1,6 +1,6 @@
+require 'pry'
 class Scrabble
```

Comment:
You have done some really cool things here, but don't sacrifice functionality for the sake of cleverness.  Get a thing done and refactor to the 'clever' solution that you want, but get it done first.